src/panel/board.ts

```diff
@@ -1,4 +1,4 @@
-import type { Container, Graphics } from "pixi.js";
+import { Container, Graphics, Point } from "pixi.js";
```

Comment:
It's useful to leave `import type` on PIXI imports, while not necessary, it prevents you from accidentally writing `new Container()` instead of `new PIXI.Container()`, which means (1) `esbuild` will pack PIXI into our output, inflating by several MB, and (2) you now have two separate instances of PIXI at runtime (generally bad) which leads to problems.

Reply:
Fixed I think this was caused by me autocompleting things